vendorsetup.sh

```diff
@@ -15,3 +15,4 @@
 #
 
 add_lunch_combo aosp_d5803-userdebug
```

Comment:
Nit: I think it's wrong since we just removed aosp_d5803 from PRODUCT_MAKEFILES.